config/kraft/README.md

```diff
@@ -35,6 +35,9 @@ Formatting /tmp/kraft-combined-logs
 
 If you are using multiple nodes, then you should run the format command on each node.  Be sure to use the same cluster ID for each one.
 
+In this example, the `server.properties` file is a `broker,controller` combined node. If you want to have a separate `broker` and `controller` node,
+Please find the example properties file [here](https://github.com/apache/kafka/blob/trunk/config/kraft/broker.properties) and [here](https://github.com/apache/kafka/blob/trunk/config/kraft/controller.properties).
+
```

Comment:
Added the last paragraph in `Format Storage Directories` section, to mention that there is also 2 other example properties file for reference.

![image](https://user-images.githubusercontent.com/43372967/146722140-ab4a48b8-3a08-4e8f-ac81-55af4854e8c6.png)